libinteract/libinteract.py

```diff
@@ -866,6 +867,166 @@ def calc_cg_fullmatrix(identifiers, idxs, percmat, perco):
     # return the full matrix (square matrix)
     return fullmatrix
 
+def find_pos(table, name = True):
+    """Takes in a table. If name is true, returns the index of the columns
+    that have the chain name. If name is false, returns the index of the
+    columns that have the res id.
+    """
+    if name:
+        # Search for SYSTEM or any chain name between A and Z
+        positions = [i for i, elem in enumerate(table[0]) \
+                     if re.search("^[A-Z]$|^SYSTEM$", elem)]
```

Comment:
I think you can just infer the position of the column from the size of the array. Or better yet, you can have arrays of the same size for all the analyses as we discussed